bin/check-influxdb-metrics.rb

```diff
@@ -79,32 +79,50 @@ class CheckInfluxDbMetrics < Sensu::Plugin::Check::CLI
          long: '--filter=VALUE',
          description: 'Set the name of your filter, for example: `datacenter`'
 
-  def encode_parameters(parameters)
-    encodedparams = Addressable::URI.escape(parameters)
-    "#{config[:db]}&q=" + encodedparams
-  end
+  option :day,
+         long: '--day=VALUE',
```

Comment:
surely it's better to pass in a period? i.e. `--period 10` or `--period 1440` and then we can adjust the query accordingly?